mpcs/src/lib.rs

```diff
@@ -377,29 +388,26 @@ pub mod test_util {
         Pcs::trim(param, poly_size).unwrap()
     }
 
-    pub fn gen_rand_poly<E: ExtensionField>(
-        num_vars: usize,
-        base: bool,
-    ) -> DenseMultilinearExtension<E> {
-        if base {
-            DenseMultilinearExtension::random(num_vars, &mut OsRng)
-        } else {
-            DenseMultilinearExtension::from_evaluations_ext_vec(
-                num_vars,
-                (0..(1 << num_vars))
-                    .map(|_| E::random(&mut OsRng))
-                    .collect_vec(),
-            )
-        }
+    pub fn gen_rand_poly_base<E: ExtensionField>(num_vars: usize) -> DenseMultilinearExtension<E> {
+        DenseMultilinearExtension::random(num_vars, &mut OsRng)
+    }
```

Comment:
in lib code (instead of testing code), we usually don't hard code RNG directly.   (besides, OsRng is slow actually, most of the time we need some kind of lightweighted rng like CHACHA RNG, that would be enough. 
  So i suggest give user  of this function the freedom to choose rng , and it is ok to use osrng in testing codes)